ci/install-bazel.sh

```diff
@@ -16,24 +16,25 @@
 
 set -eu
 
-if [[ $# -lt 1 ]]; then
-  echo "Usage: $0 <linux|osx>"
-fi
+readonly PLATFORM=$(printf "%s-%s\n" "$(uname -s)" "$(uname -m)" \
+  |  tr '[:upper:]' '[:lower:]')
 
-if [[ "$1" = "linux" ]]; then
-  readonly PLATFORM="linux-x86_64"
-elif [[ "$1" = "macos" ]] || [[ "$1" = "osx" ]]; then
-  readonly PLATFORM="darwin-x86_64"
-fi
+# Gets the latest version number of bazel from the GitHub JSON API.
+readonly BAZEL_VERSION=$(
+  curl -sL "https://api.github.com/repos/bazelbuild/bazel/releases/latest" \
+    | grep '"tag_name":' \
+    | cut -f4 -d\"
```

Comment:
Why the API? Can this result in rate-limiting? If it does, an alternative may be to parse `https://github.com/bazelbuild/bazel/releases.atom` instead.

Reply:
Alternatively, you could get the redirect URL: `URL=$(curl -Ls -o /dev/null -w '%{url_effective}' https://github.com/bazelbuild/bazel/releases/latest/)`, then use parameter substitution to get the latest part of the URL (which will be the version number: `${URL##*/}`

Reply:
Good points. While I think the rate limit of 60 requests per hour may be enough for us, it may not be and that would lead to annoying flakes in our CI build. I like your suggestions about looking at the redirect URL, however, that also seems potentially flaky to me because GitHub's web pages are for humans not scripts, so I could imagine the way they do redirects (or not) to change, and again, that may lead to flakes. 

So, for now, I think the practical thing to do is go back to using a hard-coded bazel version in our CI tests like we were before. However, I updated the version to 0.23.2. I also left in the change that compute the PLATFORM from uname.